.eslintrc.json

```diff
@@ -1,9 +1,6 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "project": "./tsconfig.*?.json"
```

Comment:
This is not strictly required for this PR, but it is a change that should be made so snuck it in!